### PR TITLE
[feature]: add docker file

### PR DESCRIPTION
**Pull request by chengmengli06**
add docker file for build tf1.15 py3 docker.

### CODE REVIEW

File: scripts/build_docker.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+bash scripts/gen_proto.sh
+if [ $? -ne 0 ]
+then
+  echo "gen proto failed"
+  exit 1
+fi
+
+sudo docker build --net=host . -f docker/Dockerfile -t  datascience-registry.cn-beijing.cr.aliyuncs.com/easyrec/easyrec:py36-tf1.15
```
Comment (tiankongdeguiji) [inline scripts/build_docker.sh:10]: tag add easy_rec version is better. py36-tf1.15-0.4.8

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
+FROM datascience-registry.cn-beijing.cr.aliyuncs.com/tensorflow/tensorflow:1.15.5
+
+COPY docker/sources_18.04.list /etc/apt/sources.list
+
+RUN cat /etc/apt/sources.list
```
Comment (tiankongdeguiji) [inline docker/Dockerfile:5]: remove

File: docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+
+RUN pip3 install pystack-debugger idna kafka-python -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN pip3 install -r /EasyRec/requirements/runtime.txt   -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN pip3 install -r /EasyRec/requirements/extra.txt   -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN pip3 install http://easyrec.oss-cn-beijing.aliyuncs.com/3rdparty/graphlearn-1.1.0-cp36-cp36m-linux_x86_64.whl -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN pip3 install http://easyrec.oss-cn-beijing.aliyuncs.com/releases/pai_automl-0.0.1rc1-py3-none-any.whl -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN pip3 install http://easyrec.oss-cn-beijing.aliyuncs.com/3rdparty/common_io-0.3.0-cp36-cp36m-linux_x86_64.whl -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN cd /EasyRec && python setup.py install
+RUN python -c "import easy_rec; easy_rec.help(); import pyhive; import datahub; import kafka"
```
Comment (tiankongdeguiji) [inline docker/Dockerfile:30]: remove and clean up /EasyRec

File: docker/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+COPY scripts /EasyRec/scripts
+
+
+RUN pip3 install pystack-debugger idna kafka-python -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN pip3 install -r /EasyRec/requirements/runtime.txt   -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN pip3 install -r /EasyRec/requirements/extra.txt   -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN pip3 install http://easyrec.oss-cn-beijing.aliyuncs.com/3rdparty/graphlearn-1.1.0-cp36-cp36m-linux_x86_64.whl -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN pip3 install http://easyrec.oss-cn-beijing.aliyuncs.com/releases/pai_automl-0.0.1rc1-py3-none-any.whl -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN pip3 install http://easyrec.oss-cn-beijing.aliyuncs.com/3rdparty/common_io-0.3.0-cp36-cp36m-linux_x86_64.whl -i http://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com
+RUN cd /EasyRec && python setup.py install
```
Comment (tiankongdeguiji) [inline docker/Dockerfile:29]: maybe pip install an easy_rec released wheel is better?

**Review by tiankongdeguiji — CHANGES_REQUESTED** [review]

File: docker/Dockerfile (hunk 4 above)
Comment (chengmengli06) [reply docker/Dockerfile:29]: build from source is more self-contained.

File: docker/Dockerfile (hunk 2 above)
Comment (chengmengli06) [reply docker/Dockerfile:5]: it is necessary to include this for later apt commands to take effect, the reason is not known

File: scripts/build_docker.sh (hunk 1 above)
Comment (chengmengli06) [reply scripts/build_docker.sh:10]: done

File: docker/Dockerfile (hunk 3 above)
Comment (chengmengli06) [reply docker/Dockerfile:30]: done

**Review by tiankongdeguiji — APPROVED** [review]
LGTM

**Pull request merged**